judgments/views/results.py

```diff
@@ -14,6 +14,7 @@
     has_filters,
     paginator,
     perform_advanced_search,
+    preprocess_query,
```

Comment:
can we add a test for the results view that captures the fact that the query now passes through `preprocess_query`?